pkg/destroy/openstack/openstack.go

```diff
@@ -1091,41 +1091,52 @@ func deleteContainers(opts *clientconfig.ClientOpts, filter Filter, logger logru
 			// Openshiftclusterid in the X-Container-Meta- HEAD output
 			titlekey := strings.Title(strings.ToLower(key))
 			if metadata[titlekey] == val {
-				logger.Debugf("Bulk deleting container %q objects", container)
-				pager := objects.List(conn, container, &objects.ListOpts{
-					Full:  false,
```

Comment:
Indeed gophercloud [defaults to `Full: false`](https://github.com/gophercloud/gophercloud/blob/v1.1.1/openstack/objectstorage/v1/objects/requests.go#L28-L31) for listing objects, it's fine to drop the explicit ListOpts.